src/main/resources/application.yaml

```diff
@@ -229,7 +229,18 @@ hapi:
     search-coord-core-pool-size: 20
     search-coord-max-pool-size: 100
     search-coord-queue-capacity: 200
-    
+
+    # Search Prefetch Thresholds.
+
+    # This setting sets the number of search results to prefetch. For example, if this list
+    # is set to [100, 1000, -1] then the server will initially load 100 results and not
+    # attempt to load more. If the user requests subsequent page(s) of results and goes
+    # past 100 results, the system will load the next 900 (up to the following threshold of 1000).
+    # The system will progressively work through these thresholds.
+    # A threshold of -1 means to load all results. Note that if the final threshold is a
+    # number other than -1, the system will never prefetch more than the given number.
+    search_prefetch_thresholds: 13,503,2003,-1
```

Comment:
If these are the default values shouldn't they be provided as such in the code here https://github.com/hapifhir/hapi-fhir-jpaserver-starter/pull/742/files#diff-79251bbeac3bb7f417a707bbda464fdfb2394f40bd88e43a9f5472e0ed351c83R107?

Reply:
Nah its fine, this is only populated if non-empty, and will rely on the HAPI-FHIR defaults if absent/empty. I put these in the properties file here for discoverability only. I hesitate to make them a java default in jpaserver-starter for fear of drift between hapi and this project.